bpf/tests/common.h

```diff
@@ -272,4 +275,22 @@ _Pragma("unroll")						\
 	return 0;						\
 }
 
+/* Asserts that the sum of per-cpu metrics map slots for a key equals count */
+#define assert_metrics_count(key, count) \
+({ \
+	struct metrics_value *__entry = NULL; \
+	__u64 sum = 0; \
+	/* Iterate until lookup encounters null when hitting cpu number */ \
+	/* Assumes at most 128 CPUS */ \
+	for (int i = 0; i < NR_CPUS; i++) { \
+		__entry = map_lookup_percpu_elem(&METRICS_MAP, &key, i); \
+		if (!__entry) { \
+			break; \
+		} \
+		sum++; \
+		break; \
```

Comment:
This `break` limits `sum` from increasing beyond 1, is it intentional?

And maybe we should `sum += __entry->count` instead of `sum++`? Otherwise sum is always set to the number of CPUs.

Reply:
Woops, fixed